native-schema-registry/python/PyGsrSerDe/KafkaSerDe.py

```diff
@@ -0,0 +1,49 @@
+from .PyGsrSerDe import *
+from .AvroSerDe import *
+
+class GlueSchemaRegistryKafkaSerializer:
+    def __init__(self):  # data format design needs discussion
```

Comment:
Do we want to add a config method for user to config the data format and naming strategy at this point

Reply:
Will implement in protobuf pr